modules/microshift-ingress-controller-config.adoc

```diff
@@ -0,0 +1,153 @@
+[source,yaml]
+----
+apiServer:
+# ...
+ingress:
+    defaultHTTPVersion: 1
+    forwardedHeaderPolicy: Append
+    httpCompression:
+        mimeTypes:
+            - ""
+    httpEmptyRequestsPolicy: Respond
+# ...
+    logEmptyRequests: Log
+# ...
+    tuningOptions:
+        clientFinTimeout: 1s
+        clientTimeout: 30s
+        headerBufferBytes: 0
+        headerBufferMaxRewriteBytes: 0
+        healthCheckInterval: 5s
+        maxConnections: 0
+        serverFinTimeout: 1s
+        serverTimeout: 30s
+        threadCount: 4
+        tlsInspectDelay: 5s
+        tunnelTimeout: 1h
+# ...
+----
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`defaultHTTPVersion`
+|`defaultHTTPVersion` sets the HTTP version for the Ingress Controller. Default value is `1`.
+//Q: which versions do we support; HTTP/1.0, 2 and 3? do we need to configure a load balancer for 2 and 3?
+
+|`forwardedHeaderPolicy`
+|By setting the `forwardedHeaderPolicy`, you specify when and how the Ingress Controller sets the `Forwarded`, `X-Forwarded-For`, `X-Forwarded-Host`, `X-Forwarded-Port`, `X-Forwarded-Proto`, and `X-Forwarded-Proto-Version` HTTP headers. The following values are valid:
+
+* `Append`, preserves any existing headers by specifying that the Ingress Controller appends them.
+* `Replace`, removes any existing headers by specifying that the Ingress Controller sets the headers.
+* `IfNone` sets the headers set by specifying that the Ingress Controller sets the headers  if they are not already set.
+* `Never`, preserves any existing headers by specifying that the Ingress Controller never sets the headers.
+
+|`httpCompression`
+|`httpCompression` defines the policy for HTTP traffic compression.
+
+* `httpCompression.mimeTypes` defines a list of or a single MIME type to which compression is applied. For example, `text/css; charset=utf-8`, `text/html`, `text/*`, `image/svg+xml`, `application/octet-stream`, `X-custom/customsub`, using the format pattern, `type/subtype; [;attribute=value]`. The `types` are: application, image, message, multipart, text, video, or a custom type prefaced by `X-`. To see the full notation for MIME types and subtypes, see link:https://datatracker.ietf.org/doc/html/rfc1341#page-7[RFC1341]
+
+|`httpEmptyRequestsPolicy`
+|`httpEmptyRequestsPolicy` describes how HTTP connections are handled if the connection times out before a request is received. Allowed values for this field are `Respond` and `Ignore`. The default value is `Respond`. The following are valid values:
+
+* `Respond`, causes the ingress controller sends an HTTP `400` or `408` response, logs the connection if access logging is enabled, and counts the connection in the appropriate metrics.
+* `Ignore`, adds the `http-ignore-probes` parameter in the `HAproxy configuration. If the field is set to `Ignore`, the ingress controller closes the connection without sending a response, then logs the connection or incrementing metrics.
```

Comment:
I think this is missing a backtick and messing up the formatting in the rendered preview. Looks like it might belong here?
```suggestion
* `Ignore`, adds the `http-ignore-probes` parameter in the `HAproxy` configuration. If the field is set to `Ignore`, the ingress controller closes the connection without sending a response, then logs the connection or incrementing metrics.
```